pkg/controller/kogitoinfra/infinispan.go

```diff
@@ -122,35 +143,114 @@ func getInfinispanAppProps(cli *client.Client, name string, namespace string) (m
 	}
 	appProps[propertiesInfinispanSpring[appPropInfinispanSaslMechanism]] = saslPlain
 	appProps[propertiesInfinispanQuarkus[appPropInfinispanSaslMechanism]] = saslPlain
+
+	if len(i.instance.Status.Volume) > 0 {
```

Comment:
what assures that the mounted volume will contain the cert info?

Reply:
This function is made to work with others in the same structure, their features are tied together. The only use case to add this volumes now is related to ISPN. I will add another check just for consistency.